dashboard-app/src/components/DataTable/DataTableHeaderRow.tsx

```diff
@@ -12,7 +12,10 @@ const HeaderRow = styled.tr`
 `;
 
 const HeaderCell = styled.th`
-  padding: ${SpacingEnum.small} 27px;
+  padding-top: ${SpacingEnum.small};
+  padding-bottom: ${SpacingEnum.small};
+  padding-left: ${SpacingEnum.small};
+  padding-right: calc(27px + calc(27px - ${SpacingEnum.small}));
```

Comment:
what's going on here?

Reply:
We want the padding between the cells to be 54px, but we also want the left padding to be 1em to match the data cells, so this just works out `54px - 1em`, which is probably what this should be changed to